pkg/admission/kubequota/kubequota_admission.go

```diff
@@ -135,8 +136,11 @@ func (k *KubeResourceQuota) Validate(ctx context.Context, a admission.Attributes
 	if a.GetResource() == tenancyv1alpha1.SchemeGroupVersion.WithResource("thisworkspaces") {
 		return nil
 	}
-	if a.GetResource() == rbacv1.SchemeGroupVersion.WithResource("clusterrolebindings") && a.GetName() == "workspace-admin" {
-		return nil
+	if a.GetResource() == rbacv1.SchemeGroupVersion.WithResource("clusterrolebindings") {
+		allowedNames := sets.NewString("workspace-admin", "workspace-admin-legacy", "workspace-access-legacy")
```

Comment:
this needs all 4

Reply:
are we missing a binding ?
```
❯ k ws "~"
Current workspace is "root:users:zu:yc:kcp-admin".
❯ k get clusterrolebindings
NAME                      ROLE                                      AGE
workspace-access-legacy   ClusterRole/system:kcp:workspace:access   15s
workspace-admin           ClusterRole/cluster-admin                 15s
workspace-admin-legacy    ClusterRole/cluster-admin                 15s
```

Reply:
nvmd. We don't create workspace-access today. A sharing feature will probably use it in the future.